compaction/compaction.cc

```diff
@@ -1275,7 +1275,7 @@ class scrub_compaction final : public regular_compaction {
     private:
```

Comment:
patch title should describe what it's being done instead, like: drop retry in compaction_stop_exception